puppet/zulip_ops/files/apt/apt.conf.d/50unattended-upgrades

```diff
@@ -1,59 +1,56 @@
 // Automatically upgrade packages from these (origin:archive) pairs
+//
+// Note that in Ubuntu security updates may pull in new dependencies
+// from non-security sources (e.g. chromium). By allowing the release
+// pocket these get automatically pulled in.
 Unattended-Upgrade::Allowed-Origins {
-//	"${distro_id}:${distro_codename}-security";
-//	"${distro_id}:${distro_codename}-updates";
-//	"${distro_id}:${distro_codename}-proposed";
-//	"${distro_id}:${distro_codename}-backports";
+    "${distro_id}:${distro_codename}";
+    "${distro_id}:${distro_codename}-security";
+    // Extended Security Maintenance; doesn't necessarily exist for
+    // every release and this system may not have it installed, but if
+    // available, the policy for updates is such that unattended-upgrades
+    // should also install from here by default.
+    "${distro_id}ESMApps:${distro_codename}-apps-security";
+    "${distro_id}ESM:${distro_codename}-infra-security";
 };
 
-// List of packages to not update (regexp are supported)
+// Python regular expressions, matching packages to exclude from upgrading
 Unattended-Upgrade::Package-Blacklist {
-//	"vim";
-//	"libc6";
-//	"libc6-dev";
-//	"libc6-i686";
+    "libc\d+";
+    "memcached$";
+    "nginx-(full|common)$";
+    "postgresql-(common|\d+)$";
+    "rabbitmq-server$";
+    "redis-server$";
+    "superivsor$";
```

Comment:
`supervisor` has a typo.

Should these patterns have a `^` at the start, if we're doing `$` at the end?

Reply:
[It uses `re.match`](https://github.com/mvo5/unattended-upgrades/blob/master/unattended-upgrade#L264), so it's not necessary.  No harm in doing so, but it's not needed.